_docs/delegators.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Delegator Overview
+subtitle: How to stake to a collator and earn rewards
+author: charles
+tags: [delegator]
+---
+
+## Risks
+*Holders of TUR tokens should perform careful due diligence on collators before delegating. Being listed as a collator is not an endorsement or recommendation from the Turing Network, or OAK Foundation. Neither the Turing Network, nor has OAK Foundation vetted the list collators and assumes no responsibility with regard to the selection, performance, security, accuracy, or use of any third-party offerings. You alone are responsible for doing your own diligence to understand the applicable fees and all risks present, including actively monitoring the activity of your collators.*
+
+*You agree and understand that neither the Turing Network, nor OAK Foundation guarantees that you will receive staking rewards and any applicable percentage provided (i) is an estimate only and not guaranteed, (ii) may change at any time and (iii) may be more or less than the actual staking rewards you receive. The OAK Foundation makes no representations as to the monetary value of any rewards at any time.*
+
+*Staking TUR tokens is not free of risk. Staked TUR tokens are locked up, and retrieving them requires a waiting period. Additionally, if a collator fails to perform required functions or acts in bad faith, a portion of their total stake can be slashed (i.e. destroyed). This includes the stake of their delegators. If a collators behaves suspiciously or is too often offline, delegators can choose to unbond from them or switch to another collator. Delegators can also mitigate risk by electing to distribute their stake across multiple collators.*
+
+## How to stake
+
+### Step 1: Figure out who you want to stake to
+
+Before staking tokens to collators, we recommend you do your research on who to stake towards. To figure out the candidate pool, you can find the set of collators in [Subscan](https://turing.subscan.io/validator). Save their wallet address as `COLLATOR_WALLET_ADDRESS` to be used on the step below.
+
+### Step 2: Figure out your inputs
+To be stake your delegator via PolakdotJS extrinsics, you'll need to figure out three numbers. We've provided a helpful script for you to use. Navigate to the **Developer > Javascript** tab on the [PolkadotJS App](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing.oak.tech#/js).
+
+```javascript
```

Comment:
Also works.
```
2a. Minimum amount to be staked: 500000000000
2b. Candidate delegation count: 0
2c. Delegation count: 0
```

Reply:
Since both collators and delegators are seeing naked tur numbers with all zeroes attached we should probably have an explainer somewhere for newbies and disclaimer about the number of 0s in tur for everyone. (Doesn't need to be included here, just something I'm assuming will come up)